py/embedded-server/deephaven_server/start_jvm.py

```diff
@@ -88,66 +95,3 @@ def start_jvm(
         config=config)
     import jpy
     jpy.VerboseExceptions.enabled = True
-
-
-def _expandLinks(dirname):
```

Comment:
these specifically existed in case a caller was specifying wildcard paths to their own extra jars

Reply:
Note: java has the mechanisms accepting wildcards natively - I don't think we should try to do the expanding ourselves if we re-introduce custom classpaths.

Reply:
Note that while java does have this mechanism from the command line, it does not work when starting the jvm as we do (at least not in my testing, which IIRC was java 11).